code/modules/mob/language/station_vr.dm

```diff
@@ -88,23 +88,26 @@
 	desc = "Shadekin seem to always know what the others are thinking. This is probably why."
 	speech_verb = "mars"
 	ask_verb = "mars"
-	exclaim_verb = "mars"
+	exclaim_verb = "MARS"
 	colour = "changeling"
 	key = "m"
 	machine_understands = FALSE
 	flags = WHITELISTED | HIVEMIND
 
-/datum/language/human/slavic
+/datum/language/slavic
 	name = LANGUAGE_SLAVIC
 	desc = "The official language of the Independent Colonial Confederation of Gilgamesh, originally established in 2122 by the short-lived United Slavic Confederation on Earth."
-	colour = "solcom"
-	key = "r"
-
+	speech_verb = "govorit" // All Russian for "says" "asks" and "shouts". Mostly here as a joke.
+	ask_verb = "voprosy"
```

Comment:
wouldn't "sprashivaet" be more fitting here (russian for 'asks')? "voprosy" means "questions". But not as in the verb, as in the noun's multiple form. So not as in "he questions" but as in "he has many questions".  For direct translation of questions as a verb, you'd use "voproshaet" but that's pretty... oldschool stuff. Something you'd see in 18th century prose, not modern day language.

Reply:
Do you speak Russian? I don't. I had to use Google and try to pick what seemed like the best word.

The only purpose of doing this is so I can even see if I am using the correct language at all.

Reply:
I do speak it as my first language. And not trying to scrutinize too much, just noticed that other three verbs were correct, while this was one that wasn't.

Reply:
Cool, so I can take your advice and feel better about it than Google translate and my own shitty understanding of Russian.